app/api/search/route.ts

```
@@ -1,4 +1,165 @@
+      },
+      body: JSON.stringify({ message: query }),
+    });
+
+    if (!response.ok) {
+      throw new Error(`Agent API error: ${response.status} ${response.statusText}`);
+    }
+
+    const data = await response.json();
+    const results = [];
+
+    if (data.answer && data.answer.trim()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)